nixos/modules/services/web-apps/immich.nix

```diff
@@ -317,6 +317,11 @@ in
       after = [ "network.target" ];
       wantedBy = [ "multi-user.target" ];
       inherit (cfg) environment;
+      path = [
+        # gzip and pg_dumpall are used by the backup service
+        pkgs.gzip
+        config.services.postgresql.package
+      ];
```

Comment:
Any reason why we don't add this to the already existing wrapper for immich? That would allow running it in contexts other than the service configured by the NixOS module.

Reply:
I wasn't sure which PostgreSQL version to use. This is a way to use the "correct" version as long as the database is local.